packages/peregrine/lib/talons/CartPage/ProductListing/useProduct.js

```diff
@@ -3,6 +3,23 @@ import { useMutation } from '@apollo/react-hooks';
 import { useAppContext } from '@magento/peregrine/lib/context/app';
 import { useCartContext } from '@magento/peregrine/lib/context/cart';
 
+/**
+ * This talon contains logic for a product component used in a product listing component.
+ * It performs effects and returns prop data for that component.
+ *
+ * @function
+ *
+ * @param {Object} props
+ * @param {ProductItem} props.item Product item data
+ * @param {ProductMutations} props.mutations GraphQL mutations for a product in a cart
+ * @param {function} props.setActiveEditItem Function for setting the actively editing item
+ * @param {function} props.setIsCartUpdating Function for setting the updating state of the cart
```

Comment:
`Function`? `function`?

Reply:
Yeah I've been inconsistent on some of the types casing due to VSCode auto-complete. I'll go back and fix.